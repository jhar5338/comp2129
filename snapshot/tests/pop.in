pop a
set a 1 2 3
get a
pop a
pop a
pop a
pop a
del a
pop a
bye
