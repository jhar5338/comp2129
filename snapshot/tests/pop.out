> no such key

> ok

> [1 2 3]

> 1

> 2

> 3

> nil

> ok

> no such key

> bye
